artifactory/utils/commandsummary/utils.go

```diff
@@ -33,3 +40,29 @@ func fileNameToSha1(fileName string) string {
 	hashBytes := hash.Sum(nil)
 	return hex.EncodeToString(hashBytes)
 }
+
+// addGitHubTrackingToUrl adds GitHub-related query parameters to a given URL if the GITHUB_WORKFLOW environment variable is set.
+func addGitHubTrackingToUrl(urlStr, section string) (string, error) {
```

Comment:
It seems that the section can only be either 'artifacts' or 'packages'. Using an enum would make it safer to manage this case.
```suggestion
func addGitHubTrackingToUrl(urlStr string, section summarySection) (string, error) {
```

```golang
type summarySection string

const (
	artifactsSection summarySection = "artifacts"
	packagesSection                 = "packages"
)
```